drivers/dac/ad5758/ad5758.h

```diff
@@ -401,30 +414,23 @@ struct ad5758_init_param {
 /******************************************************************************/
 /************************ Functions Declarations ******************************/
 /******************************************************************************/
-int32_t ad5758_set_crc(struct ad5758_dev *dev, uint8_t crc_en);
+int32_t ad5758_set_crc(struct ad5758_dev *dev, enum ad5758_crc_en crc_en);
 int32_t ad5758_wait_for_refresh_cycle(struct ad5758_dev *dev);
 int32_t ad5758_soft_reset(struct ad5758_dev *dev);
 int32_t ad5758_calib_mem_refresh(struct ad5758_dev *dev);
-int32_t ad5758_set_dc_dc_conv_mode(struct ad5758_dev *dev,
-				   enum ad5758_dc_dc_mode mode);
-int32_t ad5758_set_dc_dc_ilimit(struct ad5758_dev *dev,
-				enum ad5758_dc_dc_ilimt ilimit);
+int32_t ad5758_set_dc_dc_conv_mode(struct ad5758_dev *dev);
+int32_t ad5758_set_dc_dc_ilimit(struct ad5758_dev *dev);
 int32_t ad5758_fault_prot_switch_en(struct ad5758_dev *dev, uint8_t enable);
 int32_t ad5758_internal_buffers_en(struct ad5758_dev *dev, uint8_t enable);
-int32_t ad5758_set_out_range(struct ad5758_dev *dev,
-			     enum ad5758_output_range range);
-int32_t ad5758_slew_rate_config(struct ad5758_dev *dev,
-				enum ad5758_slew_rate_clk clk,
-				uint8_t enable);
+int32_t ad5758_set_out_range(struct ad5758_dev *dev);
+int32_t ad5758_slew_rate_config(struct ad5758_dev *dev, uint8_t enable);
```

Comment:
Why did you do this?
If use want to change the out_range for example, with this change it can't do that? what is the scope of the function then.
And no, a user shouldn't update struct ad5758_dev.

Note: Commit descriptions should include the why of the change.

Reply:
Undo-ed.